MapsRegistration.php

```diff
@@ -43,6 +43,11 @@ public static function onRegistration( array $credits ) {
 				);
 			}
 
+			// FIXME: does not work. Registration apparently too late.
+			if ( !$GLOBALS['egMapsDisableSmwIntegration'] && defined( 'SMW_VERSION' ) ) {
+				$GLOBALS['smwgNamespacesWithSemanticLinks'][NS_GEO_JSON] = true;
+			}
```

Comment:
@mwjames I'm not sure how to add this namespace to smwgNamespacesWithSemanticLinks. Apparently doing this in `wgExtensionFunctions` does not work, which is surprising to me. My guess is that registration here is too late since something already initialized the SMW code that bootstraps itself from $GLOBALS. If I do this outside of wgExtensionFunctions then it is too early, since SMW is not loaded yet (assuming you first load Maps and then SMW).

This PR works as expected when I just stuff 420 (the value of NS_GEO_JSON) into the smwgNamespacesWithSemanticLinks array in SMWs DefaultSettings.php.

Is not being able to change SMW config from wgExtensionFunctions intentional? I suspect this used to work at some point.